src/main/kotlin/com/terraformation/backend/customer/model/DeviceManagerUser.kt

```diff
@@ -31,14 +30,17 @@ data class DeviceManagerUser(
     private val parentStore: ParentStore,
     private val permissionStore: PermissionStore,
 ) : TerrawareUser {
+  companion object {
+    private const val EMAIL = "deviceManagerUser@terraformation.com"
+  }
+
   override val timeZone: ZoneId
     get() = ZoneOffset.UTC
 
   override val userType: UserType
     get() = UserType.DeviceManager
 
-  override val email: String?
-    get() = null
+  override val email = EMAIL
```

Comment:
Very slight nit: If you use `get() =` here, the generated bytecode will have a getter method that returns the singleton string constant. If you use an initializer like this, every instance of the class will have its own `email` field in memory, though other than using reflection tricks, there'll be no way to set it to a different value later on.

Reply:
Ah. I had understood getters to be not storing the value in memory which is why I intentionally avoided it, but I'm guessing that's only the case when the value isn't stored in the companion object like we're doing.

Reply:
That's it exactly. If you're doing something like `get() = BigDecimal(123)` then it would construct a new `BigDecimal` on each getter call.

String literals are kind of a special case, though. If you do `get() = "foo"` it won't construct a new `String` each time, but rather return the same object from the string pool each time.

Reply:
Hmm, I didn't mean to imply this only works because `EMAIL` is a string literal. Your comment about the companion object is correct and would apply to any kind of object, not just strings.

Reply:
Right, makes sense. That's how I interpreted your message